File: let code 1. Contains Duplicate.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let existNums = [];
        for(let num of nums)
        {
            if(existNums.includes(num))return true;
            existNums.push(num);
        };
        return false;
    }
}


// Two Sum
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let temp = new Map();
    for(let i =0; i < nums.length; i++){
        let pengurangan =  target - nums[i];
        if(temp.has(pengurangan)){
            return [temp.get(pengurangan), i]
        }
        temp.set(nums[i], i);
    }
    return [];
};

File: let code 1. Contains Duplicate.cs
public class Solution
{
	public bool hasDuplicate(int[] nums)
	{
		List<int> existNum = new List<int>();
		foreach (int num in nums)
		{
			if (existNum.Contains(num) == true)
			{
				return true;
			}
			existNum.Add(num);
		}
		return false;
	}
}


// 2. TwoSum
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int i = 0; i < nums.Length -1; i++){
            for (int j = i+1; j < nums.Length; j++){
                if(nums[i] + nums[j] == target){
                    return new int [2]{i, j};
                }
            }
        }
        return new int [2];
    }
}